### PR TITLE
Add Whimsical

### LINKED ISSUE #6746: Add Whimsical
### Site name

Whimsical

### Site URL

https://whimsical.com

### Documentation URL

https://help.whimsical.com/article/664-setting-up-two-factor-authentication-2fa

### Supported 2FA methods

- [X] TOTP (RFC-6238, Google Authenticator)
- [ ] Proprietary Software (Authy, Duo, etc.)
- [ ] FIDO2/U2F/WebAuthn security keys
- [ ] Proprietary hardware (Yubico OTP, RSA keys, etc.)
- [ ] Phone calls
- [ ] SMS tokens
- [ ] Email tokens

### Eligibility

- [X] I have checked that the site meets 2fa.directory's [contributing guidelines and site criteria](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md)
- [X] The site is ranked within the top 200,000 sites on [Similarweb](https://www.similarweb.com/) or has a substantial social media following.
- [X] The site does not contain or promote content that violates 2fa.directory's [excluded categories and websites guidelines](https://github.com/2factorauth/twofactorauth/blob/master/EXCLUSION.md), including pornographic, discriminatory, or unlawful content.
- [X] The issue I'm creating is not a duplicate of an existing [issue](https://github.com/2factorauth/twofactorauth/issues).
- [X] The issue I'm creating is not a duplicate of an existing [pull request](https://github.com/2factorauth/twofactorauth/pulls)

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by person1234-ux**
closes #6746

### CODE REVIEW

File: entries/w/whimsical.com.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "Whimsical": {
+    "domain": "whimsical.com",
+    "img" : "whimsical.png",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://help.whimsical.com/article/664-setting-up-two-factor-authentication-2fa",
```
Comment (Carlgo11) [inline entries/w/whimsical.com.json]:
```suggestion
    "documentation": "https://help.whimsical.com/article/664--",
```

File: entries/w/whimsical.com.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "Whimsical": {
+    "domain": "whimsical.com",
+    "img" : "whimsical.png",
```
Comment (Carlgo11) [inline entries/w/whimsical.com.json]:
```suggestion
    "img" : "whimsical.com.png",
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
I suggest you use this logo instead. Its file size is much more reasonable.
Please use the normal file naming scheme of `whimsical.com.png` for it.

**Comment by phallobst** [comment]
I found an SVG version of their logo in the end...
whimsical com

**Comment by person1234-ux** [comment]
I am ready for this pull request to be review and hopefully murged but of course as alway let me know if I need to make any changes before merging

**Review by Carlgo11 — APPROVED** [review]
Thanks @person1234-ux! :+1:

**Pull request merged**